lib/testUtils/getCleanOutput.mjs

```diff
@@ -0,0 +1,37 @@
+import stripAnsi from 'strip-ansi';
+
+/** @typedef {import('stylelint').Formatter} Formatter */
+/** @typedef {import('stylelint').LintResult} StylelintResult */
+/** @typedef {import('stylelint').LinterResult} LinterResult */
+
+const symbolConversions = new Map();
+
+symbolConversions.set('ℹ', 'i');
+symbolConversions.set('✔', '√');
+symbolConversions.set('⚠', '‼');
+symbolConversions.set('✖', '×');
+
+/**
+ * @param {string} output
+ * @returns {string}
+ */
+export default function getCleanOutput(output) {
+	let cleanOutput = stripAnsi(output).trim();
+
+	for (const [nix, win] of symbolConversions.entries()) {
+		cleanOutput = cleanOutput.replace(new RegExp(nix, 'g'), win);
+	}
+
+	return cleanOutput;
+}
+
+/**
+ *
+ * @param {StylelintResult[]} results
+ * @param {Formatter} formatter
+ * @param {any} returnValue
```

Comment:
I use `any` here because the default value `{ ruleMetadata: {} }` in the original `prepareFormatterOutput` is not a valid `LinterResult` (some properties are missing).

Reply:
You can use [`Omit`](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) for that.

Reply:
I wonder if there's an [`Omit`](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys)-like alternative in JSDOC.

Reply:
[suggest] Here's my suggestion. How about it?

```js
/**
 * @param {import('stylelint').LintResult[]} results
 * @param {import('stylelint').Formatter} formatter
 * @param {Pick<import('stylelint').LinterResult, 'ruleMetadata'>} [returnValue]
 * @returns {string}
 */
export function getCleanFormatterOutput(results, formatter, returnValue = { ruleMetadata: {} }) {
	// @ts-expect-error -- This error is acceptable because of the test use only.
	return getCleanOutput(formatter(results, returnValue));
}
```

Although the code increases, I believe it is more explicit and will make fewer bugs.

Reply:
@Mouvedia @ybiquitous Thank you for the suggestion! I didn't realize that I could use `Pick` and `@ts-expect-error` here. Updated here: 9780acb15cdf0f1f11a60475817f1e84ebbbdde2